src/hooks/mySql.js

```diff
@@ -32,11 +32,26 @@ function extractQueryFromArg(arg) {
   }
 }
 
+function extractValuesFromArg(arg) {
+  if (isObject(arg) && typeof arg === 'object' && !!arg['values']) {
+    return arg['values'];
+  }
+}
```

Comment:
please add an explicit return, even if it's null/undefined

Reply:
Fixed in dc7a11fbf24d18f3fa4ca6f207ead57ec93b0910